Framework/DataObjects/inc/MantidDataObjects/MDEventWorkspace.tcc

```diff
@@ -54,9 +54,8 @@ Kernel::Logger logger("MDEventWorkspace");
 TMDE(MDEventWorkspace)::MDEventWorkspace(
     Mantid::API::MDNormalization preferredNormalization,
     Mantid::API::MDNormalization preferredNormalizationHisto)
-    : API::IMDEventWorkspace(), data(nullptr),
-      m_BoxController(new API::BoxController(nd)),
-      m_displayNormalization(preferredNormalization),
+    : API::IMDEventWorkspace(), m_BoxController(new API::BoxController(nd)),
+      data(nullptr), m_displayNormalization(preferredNormalization),
```

Comment:
Note that `unique_ptr` is `nullptr` by default, so no need for `data(nullptr)`?

Reply:
Good point. I copied without spotting that.